src/Components/Footer/Footer.js

```diff
@@ -1,29 +1,54 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import "./Footer.css";
 
 export default function Footer() {
     return (
-        // TODO: add a tags to the links
+        // TODO: add a tag to Episodes when that is made
         <div className="footerContainer">
             <hr className="footerHR"></hr>
             <div className="footerLinksContainer">
                 <ul className="footerLinks">
                     <li className="footerLinksHeader">Quick Links</li>
-                    <li className="footerLink">Where to Listen</li>
+                    <Link to="/About">
+                        <li className="footerLink">Where to Listen</li>
+                    </Link>
                     <li className="footerLink">Episodes</li>
-                    <li className="footerLink">Discord Server</li>
+                    <a
```

Comment:
I am pretty sure the `<a>` tag should be a child of the `<li>` 
The only direct children in a `<ul>` or `<ol>` should be `<li>` tags

You will probably want to make that change as well with the `<Link>` tag in some of these as well